### PR TITLE
Feat/user creation button

### PR DESCRIPTION
**Pull request by sebbousquet**
Capture d’écran 2024-08-22 à 11 50 11

### CODE REVIEW

File: src/app/components/dynamic-hub/dynamic-hub.component.html [hunk 1]
```diff
@@ -17,7 +17,19 @@
     <div class="dynamic-hub__content">
         <div class="dashboards">
             <div class="dashboard" *ngFor="let data of cards | keyvalue">
-                <h2>{{data.key | titlecase | translate}} {{'dashboards' | translate}}</h2>
+                <h2>
+                    <span class="dashboard-title">{{data.key | titlecase | translate}} {{'dashboards' | translate}}</span>
+                    <ng-container *ngIf="!!canCreateDashboardByOrg.get(data.key)">
```
Comment (QuCMGisaia) [inline src/app/components/dynamic-hub/dynamic-hub.component.html:22]: you can use the getValue pipe that is declared in web components

File: src/app/components/dynamic-hub/dynamic-hub.component.html [hunk 2]
```diff
@@ -17,7 +17,19 @@
     <div class="dynamic-hub__content">
         <div class="dashboards">
             <div class="dashboard" *ngFor="let data of cards | keyvalue">
-                <h2>{{data.key | titlecase | translate}} {{'dashboards' | translate}}</h2>
+                <h2>
+                    <span class="dashboard-title">{{data.key | titlecase | translate}} {{'dashboards' | translate}}</span>
```
Comment (QuCMGisaia) [inline src/app/components/dynamic-hub/dynamic-hub.component.html:21]: we could use the key with variable substitution

File: src/app/components/dynamic-hub/dynamic-hub.component.ts [hunk 3]
```diff
@@ -82,9 +83,9 @@ export class DynamicHubComponent implements OnInit {
 
     public ngOnInit(): void {
         this.cardCollections.clear();
-        this.permissionService.get('persist/resource/').subscribe((resources: Resource[]) => {
-            this.canCreateDashboard = (resources.filter(r => r.verb === 'POST').length > 0);
-        });
+        // this.permissionService.get('persist/resource/').subscribe((resources: Resource[]) => {
```
Comment (QuCMGisaia) [inline src/app/components/dynamic-hub/dynamic-hub.component.ts:86]: to remove?

**Review by QuCMGisaia — APPROVED** [review]

File: src/app/components/dynamic-hub/dynamic-hub.component.html (hunk 1 above)
Comment (sebbousquet) [reply src/app/components/dynamic-hub/dynamic-hub.component.html]: Done

File: src/app/components/dynamic-hub/dynamic-hub.component.ts (hunk 3 above)
Comment (sebbousquet) [reply src/app/components/dynamic-hub/dynamic-hub.component.ts]: Removed

File: src/app/components/dynamic-hub/dynamic-hub.component.html (hunk 2 above)
Comment (sebbousquet) [reply src/app/components/dynamic-hub/dynamic-hub.component.html:21]: I prefer to keep it as it is for now

File: src/app/components/dynamic-hub/dynamic-hub.component.ts [hunk 4]
```diff
@@ -128,21 +123,28 @@ export class DynamicHubComponent implements OnInit {
         }
     }
 
-    public add() {
+    public add(org?: string) {
         if (this.authentMode === 'iam' && this.connected) {
+            const allowedOrganisations = [...this.canCreateDashboardByOrg.entries()].filter(m => !!m[1]).map(m => m[0]);
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply src/app/components/dynamic-hub/dynamic-hub.component.ts]: ok ! LGTM then

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**